students

```diff
@@ -1,3 +1,4 @@
 Ihor Kandaurov - https://github.com/lbrdev/Lesson1.LS.ld/
 Nastya Bezrukavaya - https://github.com/anAstyQ?tab=repositories
 Yaroslav Pavlinskiy - https://github.com/yarikpavlin/MyRepository
+Tatyana Korchinova - https://github.com/tetyanko

```

Comment:
@tetyanko link should point to your repository, but not to your profile, please update pull request.

Also:
license file should not have .md type. You can get valid license text from links in lecture.
update git.config file with execution of 'git config --list' command (there was a misprint in task description)
add some meaningful description to readme